Plugin/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ *
+ *        c.                                            c.  :.
+ *        E1                                            E.  !)
+ * ::.    E1    !3.        ,5"`'\.;F'`"t.   i.   cF'`'=.E.  !7''`   ;7""   '""!.    ;7"'"!.   ;7'`'*=
+ * ::.    E1    !3.        t.    !)     t   i.  tL      t.  !)     !)     ,,...;)  :1     I.  !t.,
+ * ::.    E1    !3.        t.    !)     t   i.  E.      E.  !)     !L    t'    :1  :1     !)    ``"1.
+ * ::.    E1    !3.        t.    !)     t   i.  '1.,  ,ct.  !1,    !L    1.  ,;31  :1     !) -..   ;7
+ * '      E1    `'         `            `   `     ``'`  `    `'``  `      `''`  `   `     `    ``'`
+ *        E7
```

Comment:
This looks good, but functionally not good to add this on every files. It can over-bloat the code, line numbers and excess file size. Please only use it on main important files, not every files.

Reply:
done